src/privatesend-server.cpp

```diff
@@ -615,7 +615,7 @@ bool CPrivateSendServer::AddEntry(const CPrivateSendEntry& entryNew, PoolMessage
 
     vecEntries.push_back(entryNew);
 
-    LogPrint("privatesend", "CPrivateSendServer::AddEntry -- adding entry\n");
+    LogPrint("privatesend", "CPrivateSendServer::AddEntry -- adding entry %d/%d\n", GetEntriesCount(), nSessionMaxParticipants);
```

Comment:
I'd prefer this log output to specify which var is which

`%d %d` - > `%d entries of %d needed entries `

Reply:
fixed